enhanced/metrics.go

```diff
@@ -626,42 +694,47 @@ var Metrics = map[string]Metric{
 			[]string{"instance", "region"},
 			map[string]string(nil),
 		),
+		Unit: 1,
 	},
 	"node_memory_SwapFree": {
 		Name: "free",
 		Desc: prometheus.NewDesc(
 			"node_memory_SwapFree",
-			"The total amount of swap memory free, in kilobytes.",
+			"The total amount of swap memory free, in bytes.",
 			[]string{"instance", "region"},
 			map[string]string(nil),
 		),
+		Unit: 1024,
 	},
 	"node_vmstat_pswpin": {
```

Comment:
hm, I am not sure, but maybe conversion is not needed here (for node_vmstat_pswpin and node_vmstat_pswpout).
please check

Reply:
`node_vmstat_pswpin` and `node_vmstat_pswpout` are delivered from `swap->in` and `swap->out` which... don't have any description at https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.html but all `swap` metrics are described to be in kilobytes.

Reply:
But we discussed it some time ago and we came with description that was saying it is in kilobytes:
https://percona.slack.com/archives/D3ZL51V0F/p1518785948000102

Reply:
agree that, it'is very unlikely be something else  instead of kilobytes

Reply:
@rnovikovP @arvenil yes, we have kilobytes in AWS.
I think we have kilobytes also in node_exporter for these metrics...
it means - maybe conversion is not needed

Reply:
node_exporter gives metrics node_vmstat_pswpin/out in pages.
So values are multiplied by page size (4096) in formulas.

Reply:
oh, so it  should be converted to  pages in RDS exporter too, for now

Reply:
@adivinho but then, we don't know in what unit is delivered metric from amazon for `node_vmstat_pswpin` and `node_vmstat_pswpout`. Those metrics are converted from `swap->in` and `swap->out` metrics from RDS, however they don't have description. On the other hand all metrics for `swap` are in kilobytes so maybe those are too.